helloWorld.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Título</title>
```

Comment:
Hey, en vez de "Título", ponle "Títulos".
```suggestion
    <title>Títulos</title>
```